lib/widget/entry_table.rb

```diff
@@ -95,13 +96,21 @@ def foot
     content_tag :tfoot do
       content_tag :tr do
         if show_result(@subject, 0) != show_result(@subject)
-          concat content_tag(:th,
-                             show_result(@subject), class: 'inner right', colspan: FIELDS.size + 1)
-          concat content_tag(:th,
-                             show_result(@subject, 0), class: 'result right')
+          concat content_tag(:th, '', colspan: FIELDS.size)
+          concat content_tag(:th) {
+            content_tag(:div,
+                         show_result(@subject), class: 'inner generic-table--footer-outer')
+          }
+          concat content_tag(:th) {
+            concat content_tag(:div,
+                                show_result(@subject, 0), class: 'result generic-table--footer-outer')
```

Comment:
Line is too long. [102/100]<br>Align the parameters of a method call if they span more than one line.